operations/tempo-mixin/alerts.libsonnet

```diff
@@ -199,7 +199,7 @@
             alert: 'TempoProvisioningTooManyWrites',
             // 30MB/s written to the WAL per ingester max
             expr: |||
-              avg by (%s) (rate(tempo_ingester_bytes_received_total{job=~".+/ingester"}[1m])) / 1024 / 1024 > 30
```

Comment:
Is it okay to just bump this to 5m, or  should it be configurable trough the config?

Reply:
The other alerts also use `5m`. I think it's fine this way.